moduleframework/environment_prepare/docker_prepare.py

```diff
@@ -31,6 +31,7 @@
 class EnvDocker(CommonFunctions):
 
     def prepare_env(self):
+        print_info('Loaded config for: {}'.format(self.config['module']['docker']['container']))
```

Comment:
if you have this line here, why to not have similar lines for nspawn and rpm? Could you please this to other module types too?